lib/apiservers/portlayer/restapi/handlers/containers_handlers.go

```diff
@@ -436,28 +428,41 @@ func convertContainerToContainerInfo(container *exec.ContainerInfo) *models.Cont
 		}
 	}
 
-	path := container.ExecConfig.Sessions[ccid].Cmd.Path
-	info.ProcessConfig.ExecPath = &path
+	// in heavily loaded environments we were seeing a panic due a missing
+	// session id in execConfig -- this has only manifest itself in short lived containers
+	// that were initilized via run
+	if _, exists := container.ExecConfig.Sessions[ccid]; exists {
+		tty := container.ExecConfig.Sessions[ccid].Tty
```

Comment:
Don't need to do `container.ExecConfig.Sessions[ccid]` again here; it has already been done on the line above.